packages/muze/src/canvas/component-wrapper-maker.js

```diff
@@ -71,6 +100,44 @@ const createHeaderWrapper = (headerType, layoutManager, renderDetails) => {
     return wrapper;
 };
 
+/**
+ * Creates the wrapper for the grid layout to be used in the tree layout
+ *
+ *
+ * @param {LayoutManager} layoutManager instance of Layout Manager which manages the layouting of the components
+ * @param {GridLayout} canvas canvas instance
+ * @return {Instance} Returns the respective wrappers
+ */
+const createMessageWrapper = (layoutManager, canvas, renderDetails) => {
+    if (!isGridPresent(renderDetails)) {
+        const defaultDimensions = { height: canvas.height(), width: canvas.width() };
+        const gridComponent = layoutManager.getComponent(GRID);
+        const { height, width } = (gridComponent && gridComponent.getBoundBox()) || defaultDimensions;
+        let messageWrapper = null;
```

Comment:
Declare the `let` statements first
Then declare the `const` statements
Have logical line spacing between const/let/process statements

Reply:
No let statements to be declared in the middle of a code block.